packages/versions/src/lib/getFuelsVersion.ts

```diff
@@ -0,0 +1,35 @@
+import fs from 'fs';
+import path from 'path';
+
+const findUserPackageJson = () => {
+  if (!require.main) {
+    return null;
+  }
+
+  let currentDir = path.dirname(require.main.filename);
+
+  while (currentDir !== path.parse(currentDir).root) {
+    const packageJsonPath = path.join(currentDir, 'package.json');
+    if (fs.existsSync(packageJsonPath)) {
+      return packageJsonPath;
+    }
+    currentDir = path.dirname(currentDir);
+  }
+
+  return null;
+};
+
+export const getLatestFuelsVersion = async () => {
+  const response = await fetch('https://registry.npmjs.org/fuels/latest');
+  const data = await response.json();
+  return data.version;
+};
```

Comment:
@Dhaiwat10 Why not use [getBuiltinVersions](https://github.com/FuelLabs/fuels-ts/blob/master/packages/versions/src/lib/getBuiltinVersions.ts) instead of a new helper?

Reply:
We need to compare both, actually:
 - last published version on NPM
 - output of `getBuiltInVersions`

This will tell us whether the user has the last version.

AFAICT, we don't need to read `package.json` files.

Reply:
@arboleya Ok, so there is no way to avoid fetching data from NPM, and therefore, dealing with a promise.

Reply:
@arboleya I was under the impression that the `FUELS` constant within the response of `getBuiltInVersions` would always be the latest release.

Reply:
@Torres-ssf that's a good point, I can use that helper to get the version of `fuels` installed by the user. But I cannot use it to get the latest version published on npm. We still need to fetch from npm for that

Reply:
https://github.com/FuelLabs/fuels-ts/pull/3108/commits/ee603f2632fcf7e90e92cd3b70836884f18d54ca

Reply:
@Dhaiwat10 @arboleya Oh I understand now. I'm sorry for the confusion.